### PR TITLE
Port Atmopsheric Network Monitoring

### PR DESCRIPTION
**Pull request by zelezniciar1**
# Description

This PR introduces the Atmospheric Network Console and gas pipe sensors, allowing Atmospheric Technicians to remotely monitor the state of pipenets. This has applications in monitoring distro pressure, supermatter cooling loop status, and locating atmospheric sabotage in the form of passive vents attached to distro. 

This will require mapping these consoles on all stations, and critically, **will require all distro, waste, and supermatter pipenets to have a gas pipe sensor added and linked to the console.**

This is a port of [Wizden #33128](https://github.com/space-wizards/space-station-14/pull/33128) and [Wizden #33294](https://github.com/space-wizards/space-station-14/pull/32294) by @chromiumboy 

tl;dr we are one step closer to fully-remote automated turing-complete atmos

---

# TODO

- [ ] Atmos Network Console
- [x] Gas Pipe Sensor
---

<details><summary><h1>Media</h1></summary>
<p>

Snímka obrazovky 2024-12-04 115117
Snímka obrazovky 2024-12-04 115203

</p>
</details>

---

# Changelog

:cl: zelezniciar
- add: Atmospheric Network Monitoring Console is now avaliable
- add: Gas Pipe Sensors are now avaliable

### CODE REVIEW

**Comment by stellar-novas** [comment]
Does https://github.com/Simple-Station/Einstein-Engines/pull/1610 achieve everything this pr adds?
